src/components/layout/ScrollFlatListScreen.tsx

```diff
@@ -28,17 +27,6 @@ interface ScreenProps<T> extends FlatListProps<T> {
 
 const ScrollFlatListScreen = <T,>({ style, onScrollEndDrag, ...props }: ScreenProps<T>) => {
   const listRef = useRef<FlatList<T>>(null)
-  const navigation = useNavigation()
-
-  useScrollToTop(listRef) // Scrolls to top when tapping the active tab
```

Comment:
This makes me think that we could implement that in the refactored DefaultHeader => a click in an empty area in the header could scroll to the top, like in the X app.

Reply:
I think that's a great idea!

Reply:
Awesome. (Merging this branch into my `UI-refinement` one will be a pain in the 🍑 😆)